.travis.yml

```diff
@@ -11,17 +11,23 @@ before_script:
   # Use Travis retry feature for farm tests since they are flaky
   - 'if [[ "$TOXENV" == "travis-test-farm"* ]]; then export TRAVIS_RETRY=travis_retry; fi'
   - export TOX_TESTENV_PASSENV=TRAVIS
+  - 'if [[ "$SNAP" == true ]]; then snap/local/build_and_install.sh; fi'
 
 # Only build pushes to the master branch, PRs, and branches beginning with
-# `test-`, `travis-test-`, or of the form `digit(s).digit(s).x`. This reduces
-# the number of simultaneous Travis runs, which speeds turnaround time on
-# review since there is a cap of on the number of simultaneous runs.
+# `test-`, `travis-test-`, or of the form `digit(s).digit(s).x` or
+# `vdigit(s).digit(s).digit(s)`. As documented at
+# https://docs.travis-ci.com/user/customizing-the-build/#safelisting-or-blocklisting-branches,
+# this includes tags so pushing tags of the form `vdigit(s).digit(s).digit(s)`
+# will also trigger tests. This reduces the number of simultaneous Travis runs,
+# which speeds turnaround time on review since there is a cap of on the number
+# of simultaneous runs.
 branches:
   # When changing these branches, please ensure the documentation under
   # "Running tests in CI" is still correct.
   only:
     - master
     - /^\d+\.\d+\.x$/
+    - /^v\d+\.\d+\.\d+$/
```

Comment:
> Could you indicate on which use case each regex refers to?

Done in commits I'll push in a bit.

> It makes me think that, for the commit that is on a branch release and is tagged for the actual release, the Travis build will be run twice.

I thought about this too, but I couldn't come up with an approach I like as I would like to continue to have the full test suite run on commits to point release branches since it automates the process of running tests before a point release. I have ideas I could describe to you, but they all have downsides that to me outweigh this optimization of saving us 1 full Travis build every release.

Unless you have a concrete suggestion, I'd prefer to ignore this inefficiency until it actually becomes annoying at which point things like other CI systems may have more options available for us.

Reply:
This is mainly relevant for our discussion at https://github.com/certbot/certbot/pull/7932#discussion_r414089205, but I wanted to note that we currently have this problem in Azure too where [advanced.yml](https://github.com/certbot/certbot/blob/4cce3458f3e6a38fbb3f39613aca1c9e1ebd86ca/.azure-pipelines/advanced.yml) is run on pushes to `*.x` branches and [release.yml](https://github.com/certbot/certbot/blob/4cce3458f3e6a38fbb3f39613aca1c9e1ebd86ca/.azure-pipelines/release.yml) is run on release tags.